api/test/config/loader.test.ts

```diff
@@ -1,3 +1,26 @@
-describe('test for test', () => {
-  expect('test').toBe('test');
+import { load } from '../../src/config/loader';
+import config from 'config';
+
+describe('Test config loader', () => {
+  it('should loading configurations from a test.yaml file is correct', () => {
```

Comment:
영어 문법에 맞게 수정해주시면 좋겠습니다.
```suggestion
  it('should load valid configurations from a test.yaml', () => {
```

Reply:
https://github.com/MovieReviewComment/Mr.C/pull/17/commits/886be2ecf6eb95767a225a2254bf4abb20e18b9a 에서 반영했습니다.